src/VolleyManagement.Backend/VolleyManagement.UI/Areas/WebAPI/Controllers/TournamentsController.cs

```diff
@@ -147,6 +150,63 @@ public ScheduleViewModel GetSchedule(int tournamentId)
                 ).ToList()
             };
 
+            if (tournament.Scheme == TournamentSchemeEnum.PlayOff)
+            {
+                ClearDivisionNames(result);
+            }
+
+            return result;
+        }
+
+        private static void ClearDivisionNames(ScheduleViewModel result)
```

Comment:
Personally for me content of this method looks strange (something like sideeffect). Have you thought about converting it to mapping, to avoid mutating (I mean creating new list rather than changing items in existing)